pytorch_toolkit/instance_segmentation/README.md

```diff
@@ -0,0 +1,263 @@
+# Instance Segmentation in PyTorch
```

Comment:
Please add `pytorch_toolkit/instance_segmentation/.gitignore`:
```
build/*
data/*
segmentoly.egg-info/*
*so
```

Reply:
@AlexanderDokuchaev `.gitignore` was added in `pytorch_toolkit/instance_segmentation` directory.